docs/conf/portland/2023/health.rst

```diff
@@ -0,0 +1,72 @@
+:template: {{year}}/generic.html
+
+Health and Safety Policy
+========================
+
+This policy is up to date as of **November 2022**.
+
+.. contents::
+    :local:
+    :depth: 1
+    :backlinks: none
+
+**Vaccine or Negative COVID-19 Test Requirement**
+-------------------------------------------------
+
+All attendees **must show proof of COVID-19 vaccination OR a negative COVID-19 test** upon first entry to the event. Attendee badges will be used as proof of vaccination or negative test validation throughout the event, each participant will need to pick up their own badge.
+
+**Vaccine Requirement**
+
+If you are showing proof of vaccination to attend in person, you will need to show that you are fully vaccinated against the COVID-19 virus. We require that you:
+
+* have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
+* that your most recent dose was at least fourteen days before the conference start
+* that your most recent dose or booster was no more than twelve months before the conference start
+
+The following are acceptable forms of proof of full vaccination:
+
+* Government-issued or government-compliant digital vaccination verification app (such as Clear or VaxYes)
```

Comment:
If possible, it would be good to add something for people with an EU Digital Covid Certificate? It's [EU+49 countries](https://ec.europa.eu/info/live-work-travel-eu/coronavirus-response/safe-covid-19-vaccines-europeans/eu-digital-covid-certificate_en) so it's pretty common. As an EU person I know I have it, but have never heard of Clear or VaxYes.

Reply:
I think that's covered by "such as" but easy to add the EU one.

Reply:
I just removed the examples? Seems easier than listing them all.